### PR TITLE
Sample Texts (Takri, Saurashtra, Hanunoo, Avestan, Kaithi, Batak, Buhid)

### PR DESCRIPTION
**Pull request by SKing-2003**
Sample text for Dogri, Takri - doi_Takr
Source: Fig. 12 in: https://www.unicode.org/L2/L2009/09111-takri.pdf (first few sentences)
Sample text for Saurashtra, Saurashtra - saz_Saur
Source: p. 3 in: http://unicode.org/L2/L2003/03256-intro-saurashtra.pdf (complete text)
Sample text for Hanunoo, Hanunoo - hnn_Hanu
Source: p.16 in: https://lingdy.aa-ken.jp/wp-content/uploads/2014/02/140227-intl-symp-and-ws_emerenciana_lorenzo_catapang_paper.pdf (complete text)
Sample Text for Avestan, Avestan - ae_Avst
Source: Fig. 6 in: https://www.unicode.org/L2/L2006/06335-n3178-avestan.pdf
Sample Text for Bhojpuri, Kaithi - bho_Kthi
Source: First couple of sentences of: https://en.wikipedia.org/wiki/Kaithi#/media/File:Kaithi2.png
Sample Text for Batak Toba, Batak - bbc_Btk
Source: First five lines of Fig 8 in: http://unicode.org/wg2/docs/n3320.pdf
Sample Text for Buhid, Buhid - bku_Bhd
Source: Poem Sample text from: https://www.omniglot.com/writing/buhid.htm

### CODE REVIEW

**Comment by RosaWagner** [comment]
Thanks @SKing-2003 
I'll wait for Simon's approval to make a release containing this PR and the previous one.

**Comment by simoncozens** [comment]
Very nice!

**Review by simoncozens — APPROVED** [review]

**Comment by simoncozens** [comment]
Rosalie: Before we release, we should get rid of the old fake text. I'll do a separate PR for that, to be merged after this one.

**Pull request merged**
